src/gui_common/SpeciesDetailsPanel.cs

```diff
@@ -38,7 +28,7 @@ public class SpeciesDetailsPanel : VBoxContainer
 
             previewSpecies = value;
 
-            if (previewSpecies != null)
+            if (previewSpecies != null && speciesPreview != null!)
```

Comment:
You should keep `speciesPreview` as nullable because that is better than having to do this dirty null override here.